src/main/java/org/waarp/common/database/DbAdmin.java

```diff
@@ -76,262 +63,262 @@
     private final DbModel dbModel;
 
     /**
-     * The connection url to be passed to the JDBC driver to establish a connection.
+     * DB Server
      */
     private final String server;
 
     /**
-     * The connection username to be passed to the JDBC driver to establish a connection.
+     * DB User
      */
     private final String user;
 
     /**
-     * The connection password to be passed to the JDBC driver to establish a connection.
+     * DB Password
      */
-    private final String password;
+    private final String passwd;
 
     /**
-     * The connection readOnly property.
+     * Is this DB Admin connected
      */
-    private boolean readOnly = false;
+    private boolean isActive = false;
+
+    /**
+     * Is this DB Admin Read Only
+     */
+    private boolean isReadOnly = false;
 
     /**
      * session is the Session object for all type of requests
      */
     private DbSession session = null;
-
     /**
      * Number of HttpSession
      */
     private static int nbHttpSession = 0;
 
-    /**
-     * Number of HttpSession
-     */
     protected static final Timer dbSessionTimer = new HashedWheelTimer(new WaarpThreadFactory("TimerClose"),
             50, TimeUnit.MILLISECONDS, 1024);
 
     /**
      * @return the session
-     *
-     * @deprecated Use {@link #getConnection()} instead.
      */
-    @Deprecated
     public DbSession getSession() {
         return session;
     }
 
-    public Connection getConnection(UUID id) throws WaarpDatabaseNoConnectionException {
-        try {
-            Connection con = ds.getConnection();
-            DbAdmin.addConnection(id, con);
-            return con;
-        } catch (SQLException e) {
-            throw new WaarpDatabaseNoConnectionException("Cannot access database", e);
-        }
-    }
-
     /**
      * @param session the session to set
      */
-    @Deprecated
     public void setSession(DbSession session) {
-        // Do Nothing
+        this.session = session;
     }
 
     /**
      * @return True if the connection is ReadOnly
      */
     public boolean isReadOnly() {
-        return readOnly;
+        return isReadOnly;
     }
 
     /**
-     * @return true
-     *
-     * @deprecated You should not use this function ask for a connection and catch errors instead.
+     * @return the isActive
      */
-    @Deprecated
     public boolean isActive() {
         return true;
     }
 
     /**
      * @param isActive the isActive to set
-     *
-     * @deprecated This will be removed in a future version of WaarpCommon. 
-     * DBCP instegration set active to true.
      */
-    @Deprecated
     public void setActive(boolean isActive) {
```

Comment:
If `isActive()` always returns `true`, `setActive()` should be a noop. 
(even the *private* property `isActive` should be removed)